doc/build-openbsd.md

```diff
@@ -1,158 +1,124 @@
 # OpenBSD build guide
-(updated for OpenBSD 5.7)
+(Tested with OpenBSD 6.6)
 
 This guide describes how to build bitcoind and command-line utilities on OpenBSD.
 
 As OpenBSD is most common as a server OS, we will not bother with the GUI.
 
+**Important**: use `gmake`, not `make`. The non-GNU `make` will exit with a horrible error.
+
 ## Preparation
 
 Run the following as root to install the base dependencies for building:
 
 ```bash
-pkg_add gmake libtool libevent
+pkg_add gmake libtool libevent git boost
 pkg_add autoconf # (select highest version, e.g. 2.69)
-pkg_add automake # (select highest version, e.g. 1.15)
-pkg_add python # (select version 2.7.x, not 3.x)
-ln -sf /usr/local/bin/python2.7 /usr/local/bin/python2
+pkg_add automake # (select highest version, e.g. 1.16)
+pkg_add python # (select highest version, e.g. 3.7)
+```
+To install openSSL, find download link here https://www.openssl.org/source/.
+
+Still as root (while modifying the version as apropriate) run
+
+```bash
+curl 'https://www.openssl.org/source/openssl-1.1.0l.tar.gz' -o openssl-1.1.0l.tar.gz
+echo '74a2f756c64fd7386a29184dc0344f4831192d61dc2481a93a4c5dd727f41148 openssl-1.1.0l.tar.gz' | sha256 -c
+# MUST output: (SHA256) openssl-1.1.0l.tar.gz: OK
+tar xvzf openssl-1.1.0l.tar.gz
+cd openssl-1.1.0l
+./config
+gmake
+gmake install
```

Comment:
why not just `gmake install` which implies `gmake` and `gmaike install`

Reply:
Brainfart on my end. Will change.

Reply:
great